src/features/all-drops/components/AllDrops.tsx

```diff
@@ -68,11 +75,54 @@ export default function AllDrops() {
   ]);
   const [wallet, setWallet] = useState({});
 
+  /** Pagination utils */
+  const [pageIndex, setPagination] = useState<number>(0); // initial page index
```

Comment:
@andrehadianto could we convert the pagination logic into a hook? where component can pass in handleGetData? and we can reuse it for drop manager page as well

Reply:
done here! https://github.com/keypom/keypom-airfoil/pull/126/commits/7cd428c679ace90313b6c46e25d91218dbf84369